src/test/java/com/thealgorithms/stacks/CelebrityFinderTest.java

```diff
@@ -0,0 +1,20 @@
+package com.thealgorithms.stacks;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class CelebrityFinderTest {
+
+    @Test
```

Comment:
Could you please refactor this to parametrized tests approach?

Reply:
Done!